apis/checkpoint/finalized_blocks/root.yaml

```diff
@@ -0,0 +1,57 @@
+get:
+  operationId: getFinalizedBlockRoot
+  summary: Get finalized block root
+  description: Retrieves hashTreeRoot of finalized BeaconBlock/BeaconBlockHeader. |
```

Comment:
What should be returned if the specified slot is an empty slot? Should it return the last block before that slot or 404?

I don't think it overly matters because you can't calculate the correct block root from a state if it's had empty slots applied, so if you have a usable state it will be from a slot that contains a block.  It's probably worth returning 404 for empty slots though just to be consistent with the `/eth/v1/beacon/blocks/{blockId}/root` API.